### PR TITLE
feat: add archive option to status menu

### PR DESCRIPTION
**Pull request by joyce-shi**
## Notion ticket link

[Archive Assessment in Display Page](https://www.notion.so/uwblueprintexecs/Archive-Assessment-in-Display-page-f66c4f87674846edb5a57ba79da87f69?pvs=4)

## Implementation description
* archive assessment in display page

## Steps to test
1. works as expected

### CODE REVIEW

File: frontend/src/components/assessments/assessment-status/EditStatusPopover.tsx [hunk 1]
```diff
@@ -73,6 +74,12 @@ const EditStatusPopover = ({
                 closePopover={onClose}
               />
             )}
+            {assessmentStatus !== Status.ARCHIVED && (
```
Comment (carissa-tang) [inline frontend/src/components/assessments/assessment-status/EditStatusPopover.tsx]: can you refactor the duplicate button to go in this block? it is also shown when status != archived. you can also do a ternary condition for the == archived and != archived

**Review by carissa-tang — COMMENTED** [review]
one more small thing! i'm not sure if all the modal texts are up to date. could you double check / confirm with design?

**Comment by joyce-shi** [comment]
> one more small thing! i'm not sure if all the modal texts are up to date. could you double check / confirm with design?

i think design mentioned that they'll change them again before we ship, so not sure we need to be too nit picky with them

File: frontend/src/components/assessments/assessment-status/EditStatusPopover.tsx [hunk 2]
```diff
@@ -13,6 +14,7 @@ import {
 import { MoreVerticalOutlineIcon } from "../../../assets/icons";
 import { Status } from "../../../types/AssessmentTypes";
 
+import ArchiveButton from "./EditStatusButtons/ArchiveButton";
```
Comment (carissa-tang) [inline frontend/src/components/assessments/assessment-status/EditStatusPopover.tsx:16]: can remove line 9 ^

**Review by carissa-tang — APPROVED** [review]

File: frontend/src/components/assessments/assessment-status/EditStatusPopover.tsx (hunk 2 above)
Comment (joyce-shi) [reply frontend/src/components/assessments/assessment-status/EditStatusPopover.tsx:16]: oops ty

**Pull request merged**